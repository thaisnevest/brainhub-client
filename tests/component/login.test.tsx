// import { render, screen } from '@testing-library/react';
// import Login from '../../src/app/login/page';
// import theme from 'styles/theme';
// import { ThemeProvider } from 'styled-components';

// describe('Login', () => {
//   it('should render the title', () => {
//     render(
//       <ThemeProvider theme={theme}>
//         <Login />
//       </ThemeProvider>
//     );
//     const title = screen.getByText('NextJS Boilerplate');
//     expect(title).toBeInTheDocument();
//     expect(title).toMatchSnapshot();
//   });

//   it('should render the subtitle', () => {
//     render(
//       <ThemeProvider theme={theme}>
//         <Login />
//       </ThemeProvider>
//     );
//     const subtitle = screen.getByText('Made with', { exact: false });
//     expect(subtitle).toBeInTheDocument();
//     expect(subtitle).toMatchSnapshot();
//   });

//   it('should render the logo', () => {
//     render(
//       <ThemeProvider theme={theme}>
//         <Login />
//       </ThemeProvider>
//     );
//     const logo = screen.getByAltText('Logo do CITi');
//     expect(logo).toBeInTheDocument();
//     expect(logo).toMatchSnapshot();
//   });
// });
