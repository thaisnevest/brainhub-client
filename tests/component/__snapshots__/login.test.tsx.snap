// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Login should render the logo 1`] = `
@media (max-width: 720px) {

}

@media (max-width: 720px) {

}

<img
  alt="Logo do CITi"
  data-nimg="1"
  decoding="async"
  height="40"
  loading="lazy"
  src="/_next/image?url=%2Fimg.jpg&w=96&q=75"
  srcset="/_next/image?url=%2Fimg.jpg&w=48&q=75 1x, /_next/image?url=%2Fimg.jpg&w=96&q=75 2x"
  style="color: transparent;"
  width="40"
/>
`;

exports[`Login should render the subtitle 1`] = `
.c0 {
  text-align: center;
  margin: 0px;
  padding: 0px;
  width: auto;
  font-family: Open Sans;
  font-weight: 400;
  font-size: 16px;
  color: #FFFFFF;
  align-self: auto;
  opacity: 1;
}

@media (max-width: 720px) {

}

@media (max-width: 720px) {

}

<p
  class="c0"
>
  Made with 
  <strong>
    &lt; / &gt;
  </strong>
   and
   
  <strong>
    â™¥
  </strong>
   by CITi
</p>
`;

exports[`Login should render the title 1`] = `
.c0 {
  text-align: center;
  margin: 20px 0;
  padding: 0px;
  width: auto;
  font-family: Outfit;
  font-weight: 800;
  font-size: 40px;
  color: #FFFFFF;
  align-self: auto;
  opacity: 1;
}

@media (max-width: 720px) {

}

@media (max-width: 720px) {

}

<p
  class="c0"
  margin="20px 0"
>
  NextJS Boilerplate
</p>
`;
